kolibri/plugins/learn/assets/src/views/HybridLearningContentCard/index.vue

```diff
@@ -84,7 +84,7 @@
   import TextTruncatorCss from 'kolibri.coreVue.components.TextTruncatorCss';
   import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
   import ProgressBar from '../ProgressBar';
-  import LearningActivityLabel from '../cards/ResourceCard/LearningActivityLabel';
+  import LearningActivityLabel from '../LearningActivityLabel';
```

Comment:
Need to update v-bind for `CardThumbnail` and `LearningActivityLabel` in here to correlate with the change to just the `contentNode` prop.

Getting vue warnings from use of this component as a result.